deploy/onpublish.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+# Travis build trigger should have environment variables COMMIT_USER, COMMIT_EMAIL and COMMIT_TOKEN setup.
+
+SOURCEFILENAME=".deploy/ddsclient-generic-gcb01.spec"
+TARGETFILENAME="ddsclient-$TRAVIS_TAG-gcb01.spec"
+COMMITMSG="DukeDSClient module update $TRAVIS_TAG"
+COMMITBRANCH="master"
+
+# Encode the spec file as base64
+BASE64CONTENT=`base64 $SOURCEFILENAME`
+
+# Upload file to helmod repo
+curl -i -X PUT -H "Authorization: token $COMMIT_TOKEN" \
```

Comment:
The DukeDSClient version number is part of the spec filename.
Since travis is doing this in response to us pushing a new version I don't know how we would end up with the spec file already existing in the helmod repo.